onnxruntime/test/providers/cpu/tensor/pad_test.cc

```diff
@@ -929,7 +929,7 @@ TEST(PadOpTest, ConstantPadAxesWithOneDimensionSpecified) {
   add support for handling negative axes.
   Issue link to the bug: https://github.com/onnx/onnx/issues/5003
 */
-TEST(PadOpTest, DISABLED_ConstantPadNegativeAxes) {
+TEST(PadOpTest, ConstantPadNegativeAxes) {
```

Comment:
cool, can you please delete the comment above on why it was disabled before?

Reply:
I'll remove the comment. The PR was merged in onnx and is part of the new release.